react/Text/Text.js

```diff
@@ -29,37 +31,35 @@ export default function Text({
   light,
   baseline,
   ...restProps
-}) {
-  return (
+}) => (
+  <span
+    {...restProps}
+    className={classnames({
+      [styles.root]: true,
+      [className]: className,
+      [styles.small]: small,
+      [styles.substandard]: substandard,
+      [styles.superstandard]: superstandard,
+      [styles.subheading]: subheading,
+      [styles.headline]: headline,
+      [styles.heading]: heading,
+      [styles.hero]: hero,
```

Comment:
Now that we have the `withTextProps` decorator creating a single `size` prop, we should probably clean this up, like you did in `Icon.js`.

Reply:
Was thinking about it, I think we could leave it like that, so we have freedom to use it as we like, and have components coded in any way as well, so it's universal both ways...thoughts?

Reply:
Sorry, not sure what you mean—might need to catch up about this one.